cli/src/clap_app.rs

```diff
@@ -84,19 +84,6 @@ pub fn get_clap_app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> A
                 .global(true)
                 .help("Show additional information"),
         )
-        .arg(
-            Arg::with_name("use_quic")
-                .long("use-quic")
-                .global(true)
-                .help("Use QUIC when sending transactions."),
-        )
-        .arg(
-            Arg::with_name("use_udp")
-                .long("use-udp")
-                .global(true)
-                .conflicts_with("use_quic")
-                .help("Use UDP when sending transactions."),
-        )
```

Comment:
we prefer to deprecate flags by hiding them, dropping their internal usage and printing a warning when they are used.  this prevents outright breaking consumers' workflow

Reply:
After some tests with this PR i have following results : 

1. It is better to send the transactions to a private RPC in RPC only mode. 
2. RPC only mode performs much worse when deploying using public testnet rpc server.

I think this is because the testnet RPC is rate limited while it is still easy to get unstaked connections on testnet.

I think it will be better to add a flag `--rpc-only` so that deploying on mainnet with private rpc will be much faster. Meanwhile still use the old quic mode by default to deploy using testnet and localcluster.

Reply:
we need to optimize the defaults for the most commonly available configuration, not what works best. more sophisticated actors can be assumed competent to adjust their configuration as appropriate.

i don't see direct submission via quic-tpu in this analysis. where does it fit in reliability? intuitively ascending reliability would be something like `unstaked_quic_tpu` -> `public_rpc` -> `private_rpc` -> `staked_tpu`. which is inverse to availability. so i'd think defaulting to `public_rpc` which can be trivially upgraded to `private_rpc` with existing `-u RPC_URL` global arg, then `--use-quic` for those with staked quic seats to further improve reliability